src/tools/pmempool/pmempool.c

```diff
@@ -32,6 +34,10 @@
 
 #define APPNAME	"pmempool"
 
+#define PMEMPOOL_LOG_PREFIX "pmempool"
+#define PMEMPOOL_LOG_LEVEL_VAR "PMEMPOOL_LOG_LEVEL"
+#define PMEMPOOL_LOG_FILE_VAR "PMEMPOOL_LOG_FILE"
```

Comment:
These variables are used by libpmempool. This means that both pmempool and libpmempool will try to write to the same file and will overwrite each other logs.